app/__init__.py

```diff
@@ -19,7 +19,7 @@
 # Register OAuth
 app.register_blueprint(ds)
 # Register examples
-app.register_blueprint(examples.eg001)
+app.register_blueprint(eg001_embedded_signing.eg001)
```

Comment:
change this back to examples

Reply:
Apparently I didn't understand how this works. Fixed. Thanks!